Change package.json so that project can run without global installation of mocha

I propose to change package.json as following.

-- added lines.
```
    "scripts": {
        "start": "node index",
        "test": "mocha test"
    },
 ...
,
    "devDependencies": {
        "mocha": "^2.5.3"
    }

```
this way project will work with following commands even mocha is not globally installed.

```
npm install
npm start
npm test
```

I already made these changes in my fork.

I don't know about jasmine; but, right now, this project package.json references mocha.
If you have no global installation of mocha, following workflow gives  error.

```
npm install
npm test

> pg-auth@1.0.0 test /your/folder/to/pg-auth
> mocha .

sh: 1: mocha: not found
npm ERR! Test failed.  See above for more details.


```

--- a/package.json
+++ b/package.json
@@ -16,7 +16,6 @@
     "license": "ISC",
     "dependencies": {
         "async": "^0.9.0",
-        "gulp": "^3.8.11",
         "massive": "^2.0.2",
         "underscore": "^1.8.2"
     },

--- a/package.json
+++ b/package.json
@@ -1,22 +1,26 @@
 {
-  "name": "pg-auth",
-  "version": "1.0.0",
-  "description": "A complete membership system built in Postgres",
-  "main": "index.js",
-  "scripts": {
-    "test": "mocha ."
-  },
-  "keywords": [
-    "postgres",
-    "membership",
-    "authentication"
-  ],
-  "author": "Rob Conery",
-  "license": "ISC",
-  "dependencies": {
-    "async": "^0.9.0",
-    "gulp": "^3.8.11",
-    "massive": "^2.0.2",
-    "underscore": "^1.8.2"
-  }
+    "name": "pg-auth",
+    "version": "1.0.0",
+    "description": "A complete membership system built in Postgres",
+    "main": "index.js",
+    "scripts": {
+        "start": "node index",
+        "test": "mocha test"
+    },
+    "keywords": [
+        "postgres",
+        "membership",
+        "authentication"
+    ],
+    "author": "Rob Conery",
+    "license": "ISC",
+    "dependencies": {
+        "async": "^0.9.0",
+        "gulp": "^3.8.11",
+        "massive": "^2.0.2",
+        "underscore": "^1.8.2"
+    },
+    "devDependencies": {
+        "mocha": "^2.5.3"
+    }
 }

